analytics/src/analytics/integrations/db.py

```diff
@@ -22,7 +22,6 @@ def get_db() -> Engine:
     A SQLAlchemy engine object representing the connection to the database.
     """
     db = get_db_settings()
-    print(f"postgresql+psycopg://{db.user}:{db.password}@{db.db_host}:{db.port}")
```

Comment:
Yeah... please don't print out the password...

Reply:
@coilysiren I agree with you. This is not my code; it was already there when I started.